build-tests/heft-jest-reporters-test/package.json

```diff
@@ -12,6 +12,7 @@
   "devDependencies": {
```

Comment:
@g-chao looks like the build is failing:

```
The shrinkwrap file (pnpm-lock.yaml) contains the following issues:
  Dependencies of project "typescript-newest-test" do not match the current shrinkwrap.
  Dependencies of project "typescript-v4-test" do not match the current shrinkwrap.
  Dependencies of project "rush-lib-test" do not match the current shrinkwrap.

  Dependencies of project "rush-sdk-test" do not match the current shrinkwrap.
  ```

Reply:
There are some bugs related to subspace. We will need to get [this PR](https://github.com/microsoft/rushstack/pull/4596) merged first.